django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -177,18 +177,21 @@ def handle(self, **options):
             destination_path = None
             message.append('.\n\n')
 
-        if self.clear:
-            message.append('This will DELETE ALL FILES in this location!\n')
-        else:
-            message.append('This will overwrite existing files!\n')
-
-        message.append(
-            'Are you sure you want to do this?\n\n'
-            "Type 'yes' to continue, or 'no' to cancel: "
-        )
+        destination_exists = (destination_path and
```

Comment:
I think it would be more clear to put this in the `if self.is_local_storage() and self.storage.location:` block